azure_arc_servers_jumpstart/docs/gcp_terraform.md

```diff
@@ -0,0 +1,123 @@
+    
+    * TF_VAR_subscription_id=Your Azure Subscription ID
+    * TF_VAR_client_id=Your Azure Service Principal app id
+    * TF_VAR_client_secret=Your Azure Service Principal password
+    * TF_VAR_tenant_id=Your Azure tenant ID
+    * TF_VAR_gcp_project_id=GCP project id
+    * TF_VAR_gcp_credentials_filename=GCP credentials json filename
+
+* From CLI, navigate to the [*azure_arc_servers_jumpstart/gcp/terraform*](../gcp/terraform) directory of the cloned repo.
+
+* Export the environment variables you edited by running [*scripts/vars.sh*](../gcp/terraform/scripts/vars.sh) with the source command as shown below. Terraform requires these to be set for the plan to execute properly. Note that this script will also be automatically executed remotely on the GCP virtual machine as part of the Terraform deployment. 
+
+    ```source ./scripts/vars.sh```
+
+* Run the ```terraform init``` command which will download the Terraform AzureRM provider.
+
+    ![](../img/gcp/08.png)
+
+* Next, run the ```terraform apply --auto-approve``` command and wait for the plan to finish. Upon completion, you will have a GCP Ubuntu VM deployed and connected as a new Azure Arc server inside a new Resource Group.
+
+# View the GCP VM from Azure Portal
+
+* Open the Azure portal and navigate to the resource group "Arc-Servers-Demo". The virtual machine created in GCP will be visible as a resource.
+
+    ![](../img/gcp/09.png)
+
+# Semi-Automated Deployment (Optional)
+
+As you may have noticed, the last step of the run is to register the VM as a new Arc server resource.
+    ![](../img/gcp/10.png)
+
+If you want to demo/control the actual registration process, do the following: 
+
+1. In the [*install_arc_agent.sh.tmpl*](../gcp/terraform/scripts/install_arc_agent.sh.tmpl) script template, comment out the "Run connect command" section and save the file.
+
+    ![](../img/gcp/11.png)
+
+2. Get the public IP of the GCP VM by running ```terraform output```
+
+    ![](../img/gcp/12.png)
+
+3. SSH the VM using the ```ssh arcadmin@x.x.x.x``` where x.x.x.x is the host ip. 
+
+    ![](../img/gcp/13.png)
+
+4. Export all the environment variables in [*vars.sh*](../gcp/terraform/scripts/vars.sh)
+
+    ![](../img/gcp/14.png)
+
+5. Run the following command
+    ```azcmagent connect --service-principal-id $TF_VAR_client_id --service-principal-secret $TF_VAR_client_secret --resource-group "Arc-Servers-Demo" --tenant-id $TF_VAR_tenant_id --location "westus2" --subscription-id $TF_VAR_subscription_id```
+
+    ![](../img/gcp/15.png)
+
+6. When complete, your VM will be registered with Azure Arc and visible in the resource group inside Azure Portal.
+
+# Delete the deployment
+
+To delete all the resources you created as part of this demo use the ```terraform destroy --auto-approve``` command.
+
```

Comment:
add a screenshot for the "terraform destroy --auto-approve"
https://github.com/likamrat/azure_arc/blob/master/azure_arc_k8s_jumpstart/docs/azure_terraform.md

Reply:
added